.vitepress/config.ts

```diff
@@ -651,6 +651,42 @@ function sidebarHome() {
                     },
                   ],
                 },
+                {
```

Comment:
can you please alphabetize the menu in this order in the PR?

1. Astria
2. Rollkit
3. Starknet
thanks!

Reply:
hi, does this work?
Astria
Dymension
Ethereum L2s:
    Arbitrum
    Optimism
    Starknet
Rollkit
Sovereign SDK
Vistara

Reply:
works for me 😄

Reply:
:shipit:

Reply:
oh sorry, let's use this organization:
hi, does this work?
Astria
Dymension
Rollkit
Sovereign SDK
Vistara

Ethereum L2s:
Arbitrum
Optimism
Starknet

Reply:
LGTM
<img width="261" alt="Screenshot 2024-03-12 at 9 42 11 AM" src="https://github.com/celestiaorg/docs/assets/46639943/390f81bb-b12b-452e-b6f6-6652482dccee">

Reply:
Hi, sorry missed these notifications; will keep the order as 
Astria, Dym, EthL2, Rollkit, Sovereign, Vistara